adoptions.md

```diff
@@ -23,17 +30,17 @@ rel="Microsoft Azure logo"><img src="/assets/images/azure_logo.jpg"
 width="200"></a>
 
 <a href="https://blog.docker.com/2015/08/content-trust-docker-1-8/" rel="docker
-logo"><img src="/assets/images/docker_logo.png" width="200"></a> 
+logo"><img src="/assets/images/docker_logo.png" width="200"></a>
 <a href="https://www.youtube.com/watch?v=2ZIu9XTvsC0"
 rel="VMware logo"><img src="/assets/images/vmware_logo.png"
-width="200"></a> 
+width="200"></a>
```

Comment:
This likely was an error.  Can you remove unintended edits like these?

Reply:
I may be missing something here but I don't see what needs fixing here.